src/Api/Tracking.php

```diff
@@ -121,4 +122,31 @@ private static function get_logins(array $params): array
 
         return $response;
     }
+
+    /**
+     * Get last tracking data filtered by days.
```

Comment:
The function DocBlock, can be make like `Retrieves the total number of published posts created in the last X days, grouped by post type.`